model/common/src/icon4py/model/common/grid/vertical.py

```diff
@@ -97,6 +103,22 @@ class VerticalGridConfig:
     #: file name containing vct_a and vct_b table
     file_path: pathlib.Path = None
 
+    # Parameters for setting up the decay function of the topographic signal for
+    # SLEVE. Default values from mo_sleve_nml.
+    #: Decay scale for large-scale topography component
+    decay_scale_1: Final[float] = 4000.0
+    #: Decay scale for small-scale topography component
+    decay_scale_2: Final[float] = 2500.0
+    #: Exponent for decay function
+    decay_exponent: Final[float] = 1.2
+    #: more constants for SLEVE coordinates
+    dvct1: Final[float] = 100.0
```

Comment:
can you come up with some good names for these? At least
```suggestion
    sleve_constant_1: Final[float] = 100.0
```